src/components/Calculator.jsx

```diff
@@ -1,80 +1,113 @@
 import React from 'react';
+import calculate from '../logic/calculate';
+import MainKeys from './common/MainKeys';
 
 class Calculator extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      obj: { total: null, next: null, operation: null },
+      enteredValue: 0,
+    };
+    this.handleSelectedValue = this.handleSelectedValue.bind(this);
+    // eslint-disable-next-line max-len
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Instead of disabling `eslint`, it will be nice to fix the issue.